deploy-manage/deploy/cloud-on-k8s.md

```diff
@@ -69,8 +69,8 @@ This section outlines the supported Kubernetes and {{stack}} versions for ECK. C
 
 ECK is compatible with the following Kubernetes distributions and related technologies:
 
-* Kubernetes 1.28-1.32
-* OpenShift 4.14-4.18
+* Kubernetes 1.29-1.33
+* OpenShift 4.15-4.19
 * Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), and Amazon Elastic Kubernetes Service (EKS)
 * Helm: {{eck_helm_minimum_version}}+
```

Comment:
because these docs are going to continue to be valid for 3.0, we need to keep the 3.0 compatibility in this doc present. 

could do it in [tabs](https://elastic.github.io/docs-builder/syntax/tabs/), table, or bulleted list. quickest hack would be: 

```suggestion
* Kubernetes: 
   * {applies_to}`eck: 3.0`: 1.28-1.32
   * {applies_to}`eck: 3.1`: 1.29-1.33
* OpenShift 
   * {applies_to}`eck: 3.0`: 4.14-4.18
   * {applies_to}`eck: 3.1`: 4.15-4.19
* Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), and Amazon Elastic Kubernetes Service (EKS)
* Helm: {{eck_helm_minimum_version}}+
```

Reply:
Any option is fine. I like both.
The list will probably not scale well when having 8-10 releases to mention, so better the tabs for that reason probably.

Reply:
Anyways, I created this issue: https://github.com/elastic/docs-builder/issues/1570

Maybe this could be a case for it.

Reply:
I think if we go with tabs, the whole list will need to go in the tabs, because it's weird to split a list between tabs and not tabs